packages/graphql-transformer-core/src/GraphQLTransform.ts

```diff
@@ -243,7 +244,7 @@ export class GraphQLTransform {
     }
 
     // check if the project is sync enabled
-    if (this.transformConfig.ResolverConfig) {
+    if (!_.isEmpty(this.transformConfig.ResolverConfig)) {
```

Comment:
This would remove CFN amplify DS resource if the blank table exists. We might not to do this in the event the table (created from the blank object) is being used for other reasons other than DataStore.